test/parallel/test-child-process-signal-validation.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
```

Comment:
We could probably get away with dropping this test and add any necessary value checks to `test-child-process-spawnsync-validation-errors.js`.